FWCore/Utilities/src/ObjectWithDict.cc

```diff
@@ -74,9 +76,13 @@ namespace edm {
     }
 
     if (to.hasBase(from)) { // down cast
+#ifndef _LIBCPP_VERSION
       // use the internal dynamic casting of the compiler (e.g. libstdc++.so)
       void* address = abi::__dynamic_cast(address_, static_cast<abi::__class_type_info const*>(&from.typeInfo()), static_cast<abi::__class_type_info const*>(&to.typeInfo()), -1);
```

Comment:
`abi::__dynamic_cast` is part of `libsupc++` in GNU world and part of `libc++abi` in LLVM world. `__dynamic_cast` in `libc++ab` is supported on Linux, Darwin and ARM systems and we don't have `libc++abi` in our LLVM toolchain builds (something that we can add).

ABI wise, I don't expect any issues as "Itanium 64-bit ABI" doesn't change, only C++ Standard Library.